framework/templates/ecl-templates-search/ecl-templates-search.twig

```diff
@@ -10,77 +10,128 @@
-            {% endembed %}
-            {% embed '@ec-europa/ecl-forms-form-groups' %}
-              {% block form_group %}
-                {% include '@ec-europa/ecl-forms-labels' with { 'label': 'Date', 'for_attribute': 'search-option-date' } %}
-                {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'date', 'id': 'search-option-date' } %}
-                  {% block options %}
-                    <option value="all">All dates</option>
-                    <option value="today">Today</option>
-                    <option value="last_week">Last week</option>
-                    <option value="last_year">Last year</option>
-                  {% endblock %}
-                {% endembed %}
-              {% endblock %}
-            {% endembed %}
-            {% embed '@ec-europa/ecl-forms-form-groups' %}
-              {% block form_group %}
-                {% include '@ec-europa/ecl-forms-labels' with { 'label': 'Language', 'for_attribute': 'search-option-language' } %}
-                {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'language', 'id': 'search-option-language' } %}
-                  {% block options %}
-                    <option value="en">English</option>
-                    <option value="de">Deutsch</option>
-                    <option value="fr">Français</option>
-                  {% endblock %}
-                {% endembed %}
-              {% endblock %}
-            {% endembed %}
-            {% embed '@ec-europa/ecl-forms-form-groups' %}
-              {% block form_group %}
-                {% include '@ec-europa/ecl-forms-labels' with { 'label': 'File Format', 'for_attribute': 'search-option-file-format' } %}
-                {% embed '@ec-europa/ecl-forms-selects' with { 'name': 'file-format', 'id': 'search-option-file-format' } %}
-                  {% block options %}
-                    <option value="all">All formats</option>
-                    <option value="doc">Doc</option>
-                    <option value="pdf">PDF</option>
-                    <option value="web">Web</option>
-                  {% endblock %}
+      <div class="ecl-u-d-flex ecl-u-d-lg-none ecl-u-justify-content-between ecl-u-align-items-baseline">
+        <h2 class="ecl-heading ecl-heading--h2">Initiatives (65)</h2>
+        {% include '@ec-europa/ecl-buttons' with {
+          'modifier': 'primary',
+          'label': 'Refine',
+          'extraClass': 'ecl-u-d-lg-none ecl-expandable__button',
+          'extraAttributes': [
+            { 'name': 'aria-controls', 'value': 'form-filter' },
+            { 'name': 'aria-expanded', 'value': 'true' },
+            { 'name': 'type', 'value': 'button' }
+          ]
+        } %}
+      </div>
+      {% embed '@ec-europa/ecl-forms-forms' with { 'extra_attributes': [{ 'name': 'aria-hidden', 'value': 'false' }, { 'name': 'method', 'value': 'get'  }, { 'name': 'method', 'value': 'get'  }, { 'name': 'id', 'value': 'form-filter' } ] } %}
```

Comment:
This will sound strange, but the `method="get" method="get"` gets added twice in the source. Browsers don't show this in the source panel, but it's visible when you look at the view-source window.